### PR TITLE
Feature/PTII-70 Add acceptance tests for 'Healthcheck Problems' page

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: tests/healthCheck_test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const assert = require('assert');
```
Comment (s4nji) [inline tests/healthCheck_test.js]: We can remove this since it's not used.

Comment (s4nji) [reply tests/healthCheck_test.js]: as well as in #253

Comment (ivanzusko) [reply tests/healthCheck_test.js]: True story! Thanx!

**Comment by s4nji** [comment]
👍

**Comment by iMartyn** [comment]
+1

**Pull request merged**